tesseract_command_language/include/tesseract_command_language/cartesian_waypoint.h

```diff
@@ -169,8 +171,18 @@ class CartesianWaypoint
   Eigen::VectorXd upper_tolerance;
 
   /**
-   * @brief Returns true if waypoint is toleranced
-   * @return True if waypoint is toleranced
+   * @brief Allow storing a seed associated with this cartesian waypoint
+   * @details Using the waypoint type erasure type to allow user defined types.
+   * The most common type that would be used it the joint waypoint for the following
+   *   - providing a seed to the IK solver
+   *   - providing a seed to the IK solver with modified limits using the tolerances
+   *   - providing a state to be used in a simple planner for interpolation to avoid running IK
+   */
```

Comment:
Minor change to the documentation

```suggestion
   * @brief Waypoint seed associated with this Cartesian waypoint
   * @details The waypoint seed can be used for purposes like:
   *   - providing a joint state seed to an IK solver
   *   - providing a seed to the IK solver with modified limits using the tolerances
   *   - providing a joint state to be used by a motion planner for interpolation to avoid performing IK
   */
```